### ISSUE TITLE
UPBGE v0.1.8b 2.78 Linux64 doesn't show Blender menus

### ISSUE THREAD

**Issue by chicortiz** [UPBGE/blender#547]
Hello UPBGE team!

I'm testing UPBGE v0.1.8b 2.78 Linux64 (stable) in  Ubuntu 14.04 and the interface buttons and menus does not appear.

Do you know what could be causing this problem?

Thank you very much for your time!

**Comment by panzergame** [comment]
Hello,

How do you get UPBGE, self-compiled or from the built releases in the server ?

**Comment by chicortiz** [comment]
Hello Panzer Game!

Thank you very much for your quick reply.

I obtained the stable release of UPBGE from here:

https://download.upbge.org/

This is how I set the options in the download page:

goo.gl/zmduJs

I also executed the recommended command using the terminal:

sudo apt-get update && sudo apt-get install blender llvm-dev libpugixml-dev

Which gave me the following log:

https://pastebin.com/xJ0Jaz4R

Greetings!

**Comment by panzergame** [comment]
A new release was uploaded to the server, could you test it please ?

**Comment by chicortiz** [comment]
Hi Panzer! 

Unfortunately it's not showing the menus on my system yet.

Take a look:

https://2.bp.blogspot.com/-V3F5thP88MQ/WYnR_zuNrSI/AAAAAAAAE-c/u-D9eKwGvHgMixR2elasRcDu_zyc4ySyACLcBGAs/s1600/Screenshot%2Bfrom%2B2017-08-08%2B11%253A56%253A47.png

**Comment by panzergame** [comment]
Could you provide the console log when launching UPBGE from console ?

**Comment by chicortiz** [comment]
The logo on terminal goes like this:

https://pastebin.com/j8En4tu9

**Comment by panzergame** [comment]
@chicortiz : Could you try to install python3.5 temporary ?

**Comment by chicortiz** [comment]
Hi guys! The menus are not showing yet
###
chico@chico-linux:~$ alias python=python3.5
chico@chico-linux:~$ python -V
Python 3.5.2
chico@chico-linux:~$ 
###

I have many python versions installed. I forgot the command to display all of my installed python versions.

Full log:
https://pastebin.com/irg5FQHY

**Comment by chicortiz** [comment]
Mmmm... Maybe I should restart the system when I set python 3.5? 
Because when I use "chico@chico-linux: $ alias python=python3.5" my regular Blender starts to behave super weird (screwed animations) xD, and the only way to set it back to normal was to do "chico@chico-linux: $ alias python=python2.7" and restart the system. I don't know how dangerous this could be, I installed the newer python versions and I'm drunk (it's Friday night over here after all).

**Comment by panzergame** [comment]
@chicortiz : Please test with recent build: https://download.upbge.org/download/release/linux64/UPBGEv0.1.9b2.78Linux64.tar.xz

**Comment by chicortiz** [comment]
Hi Panzergame! 

This version shows the menus in my sistem! :) Yay!

Thank you so much. 

Of course one can compile the UPGE, but it's always a breeze to have the option of downloading, double click and using it. 
I think this is one of the reasons the Game Engine (and Blender) is so popular: You "don't need" external dependencies. 
But (maybe) there is a problem with that: You can generate lazy users when the theme over the table is to think about how to write code (I'm just one example). 

After all,

This is just the creative mind obliterated by being forced to use menus to mediate the process. Interestingly this post is about menus. Coincidence?

I would love to know how to write code, but my mind refuses to. So I prefer to do watercolours. But one needs to wash things after doing watercolours. 

Long story short: I'm a lazy bastard, thank you so much for fixing the build.
